scripts/validation_test_harness.bash

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Checks change in a fields type
+if ! python scripts/spec_validator.py spec/validation/previous/case_one spec/validation/current/case_one ; then
```

Comment:
What do you think about changing the `case_one`,`case_two` etc folders to something that describes the test? Like `field_type_changed` for the first one